### PR TITLE
added tests for vue package

### LINKED ISSUE #13: Write tests for the vue module

### PR DESCRIPTION
**Pull request by mosheduminer**
closes #13

### CODE REVIEW

File: packages/vue/src/index.js [hunk 1]
```diff
@@ -43,7 +46,7 @@ function errorHtmlLoader(req, res, next) {
   res.locals.html = `
         <div 
             data-serveside-component="${component}" 
-            data-serveside-id="se_embed_react_ssr_${req.id}"
+            data-serveside-id="se_embed_vue_ssr_${req.id}"
```
Comment (maniator) [inline packages/vue/src/index.js:49]: Can we change all of the `se_embed_react_ssr_` in the project to something like `serveside_embed_`?

I realize that the other code is a relic

Comment (maniator) [reply packages/vue/src/index.js:49]: or maybe just `serveside_`?

File: packages/vue/src/index.js [hunk 2]
```diff
@@ -1,23 +1,26 @@
 import Vue from 'vue';
-import VueSSR from 'vue-server-renderer';
+import { createRenderer } from 'vue-server-renderer';
```
Comment (maniator) [inline packages/vue/src/index.js:2]: nice

**Review by maniator — APPROVED** [review]
Consider comments -- or I can do separately

File: packages/vue/src/index.js (hunk 1 above)
Comment (mosheduminer) [reply packages/vue/src/index.js:49]: I won't address this in this PR

Comment (mosheduminer) [reply packages/vue/src/index.js:49]: Sorry missed your comment

**Pull request merged**